From 0f69d4024aed75ee960a5ac7cf87f87d339723e1 Mon Sep 17 00:00:00 2001
From: FrancescoFerraro <francesco.f@variscite.com>
Date: Thu, 29 Sep 2022 10:28:37 +0200
Subject: [PATCH 04/12] imx8mm_var_dart: set CONFIG_SYS_LOAD_ADDR as in
 lf_v2021.04 NXP U-Boot branch

This commit fixes the following kernel error:
[    0.000000] OF: fdt: Reserved memory: failed to reserve memory for node 'vdevbuffer@0x40400000': base 0x0000000040400000, size 1 MiB

Signed-off-by: FrancescoFerraro <francesco.f@variscite.com>
(cherry picked from commit 9334c441b3c78731ea94f5bfe43fd9c852dce212)
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>
---
 configs/imx8mm_var_dart_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/imx8mm_var_dart_defconfig b/configs/imx8mm_var_dart_defconfig
index df14f704fb6..d1ef113690f 100644
--- a/configs/imx8mm_var_dart_defconfig
+++ b/configs/imx8mm_var_dart_defconfig
@@ -25,7 +25,7 @@ CONFIG_SPL_SERIAL=y
 CONFIG_SPL_DRIVERS_MISC=y
 CONFIG_SPL=y
 CONFIG_DISTRO_DEFAULTS=y
-CONFIG_SYS_LOAD_ADDR=0x40400000
+CONFIG_SYS_LOAD_ADDR=0x40480000
 CONFIG_FIT=y
 CONFIG_FIT_EXTERNAL_OFFSET=0x3000
 CONFIG_FIT_SIGNATURE=y
-- 
2.34.1

